js/company/company.js

```diff
@@ -0,0 +1,18 @@
+angular.module("modelCompany", [])
+    .factory("localModelCompany", function(){
+
+    function Company(nazwa, adres, nip, nr_konta){
```

Comment:
to co pisalem wczesniej, nazwa firmy po angielsku, a nazwy pol po polsku... bardzo brzydka praktyka :) 
koniecznie wszystko po angielsku powinno byc.